### ISSUE TITLE
Error while saving Globals - Social Poster and Blitz Installed

### ISSUE THREAD

**Issue by IndigoViking** [putyourlightson/craft-blitz#87]
### Description
```
Class 'verbb\socialposter\elements\Account' not found
in /craft/vendor/putyourlightson/craft-blitz/src/helpers/SiteUriHelper.php
```

### Steps to reproduce

1. Install Social Poster and Blitz
2. Save Global Variables
3. Error appears

### Additional info

- Plugin version: 2.0.5
- Social Poster Plugin version: 2.0.4
- Craft version: Craft Pro 3.1.18

### Error Log
`2019-03-18 22:41:47 [-][1][2bf7b7e27c600f0e25ab5faaa251a0c4][error][Error] Error: Class 'verbb\socialposter\elements\Account' not found in /home/djsclassiccars/craft/vendor/putyourlightson/craft-blitz/src/helpers/SiteUriHelper.php:68
Stack trace:
#0 /home/djsclassiccars/craft/vendor/putyourlightson/craft-blitz/src/services/ClearCacheService.php(47): putyourlightson\blitz\helpers\SiteUriHelper::getSiteSiteUris(1)
#1 /home/djsclassiccars/craft/vendor/putyourlightson/craft-blitz/src/services/RefreshCacheService.php(127): putyourlightson\blitz\services\ClearCacheService->clearSite(1)
#2 /home/djsclassiccars/craft/vendor/putyourlightson/craft-blitz/src/Blitz.php(234): putyourlightson\blitz\services\RefreshCacheService->addElement(Object(craft\elements\GlobalSet))
#3 [internal function]: putyourlightson\blitz\Blitz->putyourlightson\blitz\{closure}(Object(craft\events\ElementEvent))
#4 /home/djsclassiccars/craft/vendor/yiisoft/yii2/base/Event.php(312): call_user_func(Object(Closure), Object(craft\events\ElementEvent))
#5 /home/djsclassiccars/craft/vendor/yiisoft/yii2/base/Component.php(636): yii\base\Event::trigger('craft\\services\\...', 'afterSaveElemen...', Object(craft\events\ElementEvent))
#6 /home/djsclassiccars/craft/vendor/craftcms/cms/src/services/Elements.php(567): yii\base\Component->trigger('afterSaveElemen...', Object(craft\events\ElementEvent))
#7 /home/djsclassiccars/craft/vendor/craftcms/cms/src/controllers/GlobalsController.php(256): craft\services\Elements->saveElement(Object(craft\elements\GlobalSet))
#8 [internal function]: craft\controllers\GlobalsController->actionSaveContent()
#9 /home/djsclassiccars/craft/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#10 /home/djsclassiccars/craft/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#11 /home/djsclassiccars/craft/vendor/craftcms/cms/src/web/Controller.php(109): yii\base\Controller->runAction('save-content', Array)
#12 /home/djsclassiccars/craft/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('save-content', Array)
#13 /home/djsclassiccars/craft/vendor/craftcms/cms/src/web/Application.php(297): yii\base\Module->runAction('globals/save-co...', Array)
#14 /home/djsclassiccars/craft/vendor/craftcms/cms/src/web/Application.php(561): craft\web\Application->runAction('globals/save-co...', Array)
#15 /home/djsclassiccars/craft/vendor/craftcms/cms/src/web/Application.php(281): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#16 /home/djsclassiccars/craft/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#17 /home/djsclassiccars/public_html/index.php(21): yii\base\Application->run()
#18 {main}`

**Comment by engram-design** [comment]
FYI, I don't believe this is a Social Poster issue. For some reason its not finding registered classes, but wrapping https://github.com/putyourlightson/craft-blitz/blob/v2/src/helpers/SiteUriHelper.php#L68 with `if (class_exists($elementType)) {` should help.

I wonder if it maybe has to do with the order plugins/classes are loaded at the time this code is run, hence why it can't find the class? Local testing on my end has this happen with a number of other plugins.

**Comment by putyourlightson** [comment]
@engram-design Where exactly is the _Account_ element? I'm not seeing it where I would expect to:
https://github.com/verbb/social-poster/tree/2.0.4/src/elements

**Comment by engram-design** [comment]
Well that's embarrassing. Fixed in [2.0.6](https://github.com/verbb/social-poster/releases/tag/2.0.6). You win this round @putyourlightson 😁
